sipi/scripts/sipi.init.lua

```diff
@@ -20,14 +20,16 @@ require "get_knora_session"
 --    filepath: server-path where the master file is located
 -------------------------------------------------------------------------------
 function pre_flight(prefix, identifier, cookie)
-    server.log("pre_flight called in sipi.init.lua", server.loglevel.LOG_DEBUG)
+    server.log(os.date("!%Y-%m-%dT%H:%M:%S") .. " - pre_flight called in sipi.init.lua", server.loglevel.LOG_DEBUG)
```

Comment:
As you are repeating the same pattern for the timestamp, could it be set as a global pattern?

Reply:
done now

Reply:
done